client/packages/invoices/src/OutboundShipment/DetailView/columns.ts

```diff
@@ -218,7 +218,7 @@ export const useOutboundColumns = ({
       },
     ],
     {
-      label: 'label.unit-price',
+      label: 'label.sell-price',
```

Comment:
I think including a description that this is per unit would be helpful, e.g. this example:

Pack sell price is $5.50:

![Screenshot 2025-01-07 at 4 43 43 PM](https://github.com/user-attachments/assets/1b71e827-e6f3-46a1-9622-a6ebddc44001)